### PR TITLE
Updated comments in rule files on how to use regexp-assemble.py

### PR DESCRIPTION
**Pull request by theseion**
This PR accompanies #2422.

### CODE REVIEW

**Comment by airween** [comment]
I hope you will continue to unify the regexp-assembly script documentation :).

Now I'm using this regex to catch the build instruction in the CRS Rules-doc script:
```
        self.re_buildinst = re.compile("To rebuild the (regexp|regular expression)")
```

It would be good to see that each description uses the same syntax. Thanks :).

File: rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf [hunk 1]
```diff
@@ -103,7 +103,7 @@ SecRule REQUEST_LINE "!@rx (?i)^(?:(?:[a-z]{3,10}\s+(?:\w{3,7}?://[\w\-\./]*(?::
 #
 # To rebuild the regular expression:
```
Comment (RedXanadu) [inline rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf:104]: rebuil -> update?

File: rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf [hunk 2]
```diff
@@ -85,7 +85,7 @@ SecRule TX:EXECUTING_PARANOIA_LEVEL "@lt 1" "id:932012,phase:2,pass,nolog,skipAf
 #
 # To rebuild the word list regexp:
```
Comment (RedXanadu) [inline rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf:86]: rebuild -> update?

**Comment by theseion** [comment]
Thanks guys! I've changed the comment a bit but I think I got them all this time.

I don't think there's any merit in having "word list" in two places, it doesn't add any useful information, IMO.

**Comment by franbuehler** [comment]
[Meeting decision March 7](https://github.com/coreruleset/coreruleset/issues/2401#issuecomment-1061076993): @RedXanadu already reviewed. @franbuehler and @airween are added as reviewers.

**Review by airween — APPROVED** [review]
I reviewed this PR, tried with the parser of the CRS documentation (which is also aligned).

Looks good to me.

**Comment by dune73** [comment]
We discussed this in the previous meeting:

> #2423 depends on https://github.com/coreruleset/coreruleset/pull/2422 that still needs a review by @franbuehler and @airween.

Adding do-not-merge label.

**Comment by dune73** [comment]
Removing the do-not-merge label since #2422 has been merged in the meantime.

The merging of this PR is critical for the v.4-RC.

**Comment by airween** [comment]
> Removing the do-not-merge label since #2422 has been merged in the meantime.

The script which builds the CRSDoc structure uses heavily these comments. Let me check the modifications with that before the merge.

**Comment by dune73** [comment]
Very good. Thanks.

**Comment by theseion** [comment]
> > Removing the do-not-merge label since #2422 has been merged in the meantime.
> 
> The script which builds the CRSDoc structure uses heavily these comments. Let me check the modifications with that before the merge.

Thanks, I wasn't aware of that.

**Comment by airween** [comment]
The modified rules are fine.

But I found few rules where the build instructions are in some old format (what I don't handle either in the crsdoc script).

@theseion do you think you can change these lines too?

[932300](https://github.com/theseion/coreruleset/blob/unified-regex-utils-update-comments/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf#L746-L748)
[932310](https://github.com/theseion/coreruleset/blob/unified-regex-utils-update-comments/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf#L778-L780)
[932320](https://github.com/theseion/coreruleset/blob/unified-regex-utils-update-comments/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf#L812-L814)
[932301](https://github.com/theseion/coreruleset/blob/unified-regex-utils-update-comments/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf#L922-L924)
[932311](https://github.com/theseion/coreruleset/blob/unified-regex-utils-update-comments/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf#L955-L957)
[932321](https://github.com/theseion/coreruleset/blob/unified-regex-utils-update-comments/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf#L988-L990)

Thanks!

**Comment by theseion** [comment]
Thanks @airween, nicely spotted. I also found another lingering one (934120). I hope I got them all now.

**Comment by airween** [comment]
> I hope I got them all now.

Sorry, after I grabbed you commit, I found a new one occurrence:

[930100](https://github.com/coreruleset/coreruleset/blob/v3.4/dev/rules/REQUEST-930-APPLICATION-ATTACK-LFI.conf#L30-L35).

**Comment by theseion** [comment]
That one has to stay that way for now. There's no good way at the moment to do it differently. I'm working on some enhancements to regexp-assemble that will take care of that.

**Pull request merged**